README.md

```diff
@@ -1,2 +1,43 @@
 # lux-benchmark
 Experiments to benchmark Lux Performance
+
+## Setup
+First, ensure `lux-datasets` is in the same directory as this one locally. This will allow you to run the following command to generate large airbnb and communities datasets:
```

Comment:
We could add lines to download (e.g., wget) the [airbnb](https://github.com/lux-org/lux-datasets/blob/master/data/airbnb_nyc.csv) and communities dataset from lux-datasets to pull the data into the `data/` directory.  We should also add what the directory structure should look like and where the make_dataset script needs to be run from (e.g., root).